cms/jinja2/templates/pages/statistical_article_page.html

```diff
@@ -191,7 +191,10 @@ <h2 class="headline-figures__heading ons-u-fs-m">{{ _("Headline facts and figure
                     onsTableOfContents({
                         "title": toc_title,
                         "ariaLabel": toc_aria_label,
-                        "itemsList": table_of_contents
+                        "itemsList": table_of_contents,
+                        "button": {
+                            "text": _("Save or print this page")
+                        }
```

Comment:
q: We output an "Explore data" link below which still uses the TOC component (https://github.com/ONSdigital/dis-wagtail/blob/dad10d9407dde5c84d8c5c5f9ef2f24b5533c344/cms/jinja2/templates/pages/statistical_article_page.html#L203-L220), could this be incorporated?

The change came in #165

Reply:
Ah brilliant thanks for flagging this @zerolab - updated now. Do we have a similar update coming for the Topic pages?

Reply:
We do. Will look at making a note on the relevant ticket about the FE approach